pkg/trace/config/config.go

```diff
@@ -53,7 +53,14 @@ type OTLP struct {
 	// automatically map Datadog Span Operation Names to an updated value. All entries should be key/value pairs.
 	SpanNameRemappings map[string]string `mapstructure:"span_name_remappings"`
 
-	// SpanNameAsResourceName uses the OTLP span name as the Datadog resource name.
+	// SpanNameAsResourceName specifies whether the OpenTelemetry span's name should be
+	// used as the Datadog span's operation name. By default (when this is false), the
+	// operation name is deduced from a combination between the instrumentation scope
+	// name and the span kind.
+	//
+	// For context, the OpenTelemetry Span Name's equivalent at Datadog is the resource name.
```

Comment:
The phrasing here took me a little bit to understand, is this a bit clearer you think?
```suggestion
	// For context, the OpenTelemetry 'Span Name' is equivalent to the Datadog 'resource name'.
```